core/src/main/java/dk/alexandra/fresco/lib/math/bool/log/Logarithm.java

```diff
@@ -80,9 +80,6 @@ public Logarithm(List<DRes<SBool>> number) {
    * Computes the floor(log_2(x)).
    */
   private int log2(int n) {
-    if (n <= 0) {
```

Comment:
Why delete this check?

Reply:
It's a private method and could never be invoked with n<=0